cmd/operator-sdk/bundle/validate.go

```diff
@@ -150,27 +134,61 @@ func (c bundleValidateCmd) validate(args []string) error {
 // exit code to be returned (true by default).
 func (c *bundleValidateCmd) addToFlagSet(fs *pflag.FlagSet) {
 	fs.StringVarP(&c.imageBuilder, "image-builder", "b", "docker",
-		"Tool to extract bundle image data. Only used when validating a bundle image. "+
-			"One of: [docker, podman]")
+		"Tool to pull and unpack bundle images. Only used when validating a bundle image. "+
+			"One of: [docker, podman, none]")
+
 	fs.StringVarP(&c.outputFormat, "output", "o", internal.Text,
```

Comment:
when we call the bundle validation with the --image-builder and then bundle is in the disk (created with the generated-only) it is not WARNING that the value will be ignored or raising and error telling that the flag is invalid. Could we add that?

Reply:
We should either fail if that flag is set or do nothing, but that's something to do in a follow-up.